SU-SWS/decanter
# Webpack Revisions: Integration Improvements

# READY FOR REVIEW

# Summary
- I ran into troubles getting this to build with my Drupal theme so I needed a few changes. 
- Separate the JS, SASS, and KSS builds completely. 
- Change entry points to the SCSS files instead of having empty or dummy JS files. 
 - 

When trying to import and build using version 5 I ran into an issue with the JS file and the CSS files. I was not able to include and build them into my project independently. It may be possible to do what I want with splitchunks but I would rather keep the two things separate. 

# Needed By (Date)
- The sooner you can look at this the sooner it will help me.

# Urgency
- Not critical. 

# Steps to Test

1. Check out this branch
2. Throw out your `node_modules` folder
3. Run `npm install`
4. Run `npm run build` and ensure everything builds without error
5. Run `npm run styleguide` and `npm run start` and ensure the styleguide looks right
6. Read through the code and leave comments on the changes.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
